File: remidial - 2/Poll/Quiz.java
class Quiz{

	Question[]list;
	int listsize;

	public Quiz(){
		list = new Question[10];
		listsize = 0;

	}
	public void addQuestion(Question question){
		for (int i = 0;i <list.length;i++) {
			if (list[i]==null) {
				list[i] = question;
				listsize++;
				break;
			}
			
		}

	}
	public int size(){
		return listsize;
	}
	public Question getQuestion(int quesno){
		 return list[quesno];

	}
}

File: remidial - 2/my sol/Poll/Quiz.py

from Question import *
class Quiz(object):
	"""docstring for Quiz"""
	def __init__(self):
		self.qlist = []
		self.qlsize = 0

	def addQuestion(self,question):
			self.qlist.append(question)
	def getQuestion(self,quesno):
		return self.qlist[quesno]



		

